### PR TITLE
ONYX-10851 Mapping claims naming alignment

### PR DESCRIPTION
**Pull request by sashaCher**
### What does this PR do?

Align class names according "mapping-claims" variable name.

### What ticket does this PR close?
ONYX-10851

#### Change log
- [ ] The CHANGELOG has been updated, or
- [X] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [X] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [X] This PR does not require updating any documentation

#### API Changes
- [ ] The [OpenAPI spec](https://github.com/cyberark/conjur-openapi-spec) has been updated to meet new API changes (or an issue has been opened), or
- [X] The changes in this PR do not affect the Conjur API

### CODE REVIEW

**Review by tzheleznyak — DISMISSED** [review]
LGTM

File: app/domain/authentication/authn_jwt/consts.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@ module AuthnJwt
     IDENTITY_PATH_DEFAULT_VALUE = ""
     IDENTITY_PATH_CHARACTER_DELIMITER = "/"
     IDENTITY_TYPE_HOST = "host"
-    MANDATORY_CLAIMS_RESOURCE_NAME = "mandatory-claims"
+    MANDATORY_CLAIMS_RESOURCE_NAME = "host-mandatory-claims"
```
Comment (nessiLahav) [inline app/domain/authentication/authn_jwt/consts.rb:18]: @sashaCher @hilagross when it was changed ?

Comment (sashaCher) [reply app/domain/authentication/authn_jwt/consts.rb]: ¯\_(ツ)_/¯

**Review by nessiLahav — APPROVED** [review]
LGTM

**Pull request merged**